packit/jobs.py

```diff
@@ -298,6 +298,19 @@ def run(self):
         )
 
 
+# @add_to_mapping
+# class CoprBuildFinished(FedmsgHandler):
+#     topic="org.fedoraproject.prod.copr.build.end"
+#     name = JobType.ReportCoprResult
+#
+#     def run(self):
+#         msg = f"Build {self.event['msg']['build']} " \
+#               f"{'passed' if self.event['msg']['status'] else 'failed'}.\n" \
+#               f"\tpackage: {self.event['msg']['pkg']}\n" \
+#               f"\tchroot: {self.event['msg']['chroot']}\n"
+#         # TODO: lookup specific commit related to the build and comment on it
```

Comment:
can we actually store some metadata in the copr build? PR ID, github repo would be really useful, commit ID as well